languages.toml

```
@@ -1409,3 +1409,16 @@ injection-regex = "verilog"
 [[grammar]]
 name = "verilog"
 source = { git = "https://github.com/andreytkachenko/tree-sitter-verilog", rev = "514d8d70593d29ef3ef667fa6b0e504ae7c977e3" }
+
+[[language]]
+name = "nim"
+scope = "source.nim"
+file-types = ["nim", "nimble"]
```

```suggestion
file-types = ["nim", "nims", "nimble"]
```

to add support for nimscript